index.js

```diff
@@ -62,7 +62,7 @@ module.exports = request
     if (err.response) {
       logger.info(err.response.json || err.response.text);
     }
-    process.exit(err.status || 2); // eslint-disable-line no-process-exit
+    process.exit(config.failOnError ? err.status || 2 : 0); // eslint-disable-line no-process-exit
```

Comment:
Will this exit with zero, no matter what, whenever the flag is set? That doesn't seem to be quite right. (Though maybe it's good enough for now.)

Ideally we want it to exit zero no matter what the API says. But we still want other exit codes for any other problems.

Then again, are there any such error conditions?